src/main/java/xonin/backhand/client/ClientEventHandler.java

```diff
@@ -37,12 +39,15 @@
 @EventBusSubscriber(side = Side.CLIENT)
 public class ClientEventHandler {
 
+    private static final ResourceLocation OFFHAND_SLOT_TEXTURE = new ResourceLocation(
+        Backhand.MODID,
+        "textures/gui/offhand_slot.png");
     public static boolean prevInvTweaksAutoRefill;
     public static boolean prevInvTweaksBreakRefill;
     public static int invTweaksDelay;
 
     @SubscribeEvent
-    public static void renderHotbarOverlay(RenderGameOverlayEvent event) {
+    public static void renderHotbarOverlay(RenderGameOverlayEvent.Post event) {
```

Comment:
it is possible that a mod cancels the pre HOTBAR even, which means your handler won't run, for compatibility it would be better to subscribe to the `PRE` event and use `@SubscribeEvent(receiveCanceled = true)`